pootle/apps/pootle_project/models.py

```diff
@@ -402,12 +402,14 @@ def get_cachekey(self):
     # # # /TreeItem
 
     def get_stats_for_user(self, user):
-        self.set_children(self.get_children_for_user(user))
+        self.set_children(
+            self.get_children_for_user(
+                user, select_related=["language__name", "directory"]))
```

Comment:
@phlax, you wrote: "i dont think you can do select_related("fk__otherfield")"...